helm/postgres/values.yaml

```diff
@@ -47,7 +47,7 @@ postgresVersion: 14
 # below value. "postgresVersion" needs to match the version of Postgres that is
 # used here. If using the GIS-enabled Postgres image, you need to ensure
 # "postGISVersion" matches the version of PostGIS used.
-# imagePostgres: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.7-0
+# imagePostgres: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-15.1-0
```

Comment:
should this be `15.2`?

Reply:
updated all the images to the latest versions